api/provider/aws/environment_update.go

```diff
@@ -6,6 +6,10 @@ import (
 	"github.com/quintilesims/layer0/common/models"
 )
 
+// Update updates an ECS Cluster using the specified Update Environment Request.
```

Comment:
[nit] same here.
"Update the ECS Cluster using the specified Update Environment Request."